src/order/well_founded.lean

```diff
@@ -20,6 +20,25 @@ and provide a few new definitions: `well_founded.min`, `well_founded.sup`, and `
 variables {α : Type*}
```

Comment:
Although maybe this should be
```suggestion
variables {α : Sort*}
```

Reply:
This makes all of the statements about partial orders, order duals, etc, error out.

Reply:
In that case maybe worth adding a
```suggestion
variables {α : Type*} {α' : Sort*}
```
up here (or beta) and using it in the new lemmas below that _do_ work for `Sort`

Reply:
Actually nevermind, I see that `{α}` is pretty common throughout this file. Sorry for the distraction